packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -48,6 +48,38 @@ const ErrorAfter4s = () => {
 
 ![Example banner](/gif/errorboundary-example.gif)
 
+:::tip
+
+## Use `<ErrorBoundary/>` fallback props without prop drilling
+
+### useErrorBoundaryFallbackProps
+
+In fallback of `<ErrorBoundary/>`, by calling useErrorBoundaryFallbackProps().reset/error, we can use error of `<ErrorBoundary/>` and reset it without using prop directly
+
+```tsx
+import { ErrorBoundary, useErrorBoundaryFallbackProps } from '@suspensive/react'
+
+const Nested = () => {
+  const props = useErrorBoundaryFallbackProps()
+
+  return (
+    <>
+      <button onClick={props.reset} />
+      {props.error.message}
+    </>
+  )
+}
+
+const ErrorBoundaryFallback = () => <Nested />
+const Example = () => (
+  <ErrorBoundary fallback={ErrorBoundaryFallback}>
```

Comment:
How about add onReset prop ??

User who show this example or document first time, just draged whole example and paste their code. and curious "why my query is not reseted and don't query again?"

Reply:
> and curious "why my query is not reseted and don't query again?"

I guess `query` you mentioned is for @tanstack/react-query. but because @suspensive/react is not only for @tanstack/react-query, I think we should not to describe this example with @tanstack/react-query

Reply:
I agree, thanks!